source/shared/cpp/ObjectModel/CollectionTypeElement.h

```diff
@@ -109,8 +109,12 @@ namespace AdaptiveSharedNamespace
 
         collection->SetBleed(ParseUtil::GetBool(value, AdaptiveCardSchemaKey::Bleed, false));
 
-        collection->SetMinHeight(
-            ParseSizeForPixelSize(ParseUtil::GetString(value, AdaptiveCardSchemaKey::MinHeight), &context.warnings));
+        if (const auto& minHeight =
```

Comment:
this is cool; i didn't know we could do this. thanks.

Reply:
it's a C++17 feature :)

https://en.cppreference.com/w/cpp/language/if
(see "if statements with initializer")

Reply:
love it!